package.json

```diff
@@ -17,7 +17,8 @@
     "typescript": "5.5.4"
   },
   "workspaces": [
-    "sdk"
+    "sdk/tbrv3/evm",
```

Comment:
do we need the `tbrv3` directory here? ultimately this is the only contract we'll have in this repo, isn't it?

Reply:
For now, yeah. But I'm expecting the TBRv3 sdk to outlive the TBRv3 contract and I would like to host all sdks here :thinking:

Reply:
gotcha, thanks

Reply:
It's done as discussed